mount/mount_windows_test.go

```diff
@@ -133,6 +134,10 @@ func TestPathWithinBase(t *testing.T) {
 }
 
 func TestIsLikelyNotMountPoint(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		// symlink test need priviledge access, skip now
+		t.Skip("Skipping test on Windows")
```

Comment:
what privilege it requies?

Reply:
This file is already scoped to windows `// +build windows`, this is effectively skipping this test 100% of the time. Doesn't that leave this method untested?

Reply:
point is this `TestIsLikelyNotMountPoint` func is not tested in any windows unit test CI in this repo, we cannot test it in Github action on Windows since mklink need admin priviledge, while we don't know how to get admin priviledge in github action now, otherwise cannot pass on Windows UT in GH action.

Reply:
my point is Windows unit tests are never running on CI, it's better enable windows CI first.

Reply:
/hold
hold on, there is one way to only skip one unit test in Github action, that would be better

Reply:
do you know any other commands need admin priviledge? What if we use powershell command like below to create a link?
```
New-Item -Path C:\LinkDir -ItemType SymbolicLink -Value F:\RealDir
```

Reply:
It's the same error:
```
PS C:\Users\xiazhang> New-Item -Path C:\tmp\LinkDir -ItemType SymbolicLink -Value c:\tmp
New-Item : Administrator privilege required for this operation.
At line:1 char:1
+ New-Item -Path C:\tmp\LinkDir -ItemType SymbolicLink -Value c:\tmp
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (C:\tmp:String) [New-Item], UnauthorizedAccessException
    + FullyQualifiedErrorId : NewItemSymbolicLinkElevationRequired,Microsoft.PowerShell.Commands.NewItemCommand
```

Reply:
/hold cancel
PTAL, I used an env var to skip this test in github action.

Reply:
found it mentioned about this issue https://github.com/MicrosoftDocs/PowerShell-Docs/pull/3473/files